src/api/comments.js

```diff
@@ -0,0 +1,24 @@
+import { BASE_URL } from './api';
+
+export const getPostComments = async(postId) => {
+  const postComments = await fetch(`${BASE_URL}/comments/`);
```

Comment:
Now you are loading all comments. It's a lot. You should load only comments that are related to a specific post. Use the endpoint:
`/comments?postId=${postId}` for fetching only comments for specified postId